Configuration/Services.yaml

```diff
@@ -131,3 +138,6 @@ services:
   StefanFroemken\ExtKickstarter\Service\Creator\UpgradeWizardCreatorService:
     arguments:
       $upgradeCreators: !tagged_iterator { tag: 'ext-kickstarter.creator.upgrade-wizard' }
+  StefanFroemken\ExtKickstarter\Service\Creator\ValidatorCreatorService:
```

Comment:
This list is ordered by tag-name

Reply:
so V is after U?

Reply:
Yes, like you have done in Services.php, too ;-) If you want to keep it at bottom feel free to order all command registrations by Classname, but in that case please order the values in Services.php, too, to have them symmetrical

Reply:
ah, TAG not class name, ic